CWE-94-Python_darwin/dali-lab_RoomKitServer/ML.py

```
@@ -0,0 +1,67 @@
+
+def model_to_str(model):
+    return pickle.dumps(model)
+
+
+def load_model(string):
+    """
+    Loads the model from a string
+    :param string: The string
+    :return: MLPClassifier
+    """
+    return pickle.loads(string)
```

Ruff (0.13.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)